### DISCUSSION TITLE
Global Delay Option for Kit

### DISCUSSION THREAD

**Discussion by Akilestar** [EssentialsX/Essentials#3857] [Ideas and Feature Suggestions] [answered]
I'm using kits to spawn boats in a "shipyard" and attaching the kits to signs so players can grab a boat and set sail. I have several places for the boats to spawn but it would be nice if a player spawns a boat using the sign for another player to be unable to use the same sign for a period of time. The easiest solution would be for a global delay so if a kit has been used by anyone on the server, no one could use that kit until the cooldown period has passed. 

Thanks for the great mod and everything it does for the community.

**Comment by mdcfe** [comment, marked as answer]
This seems like a very specific and niche use case, and I'm not sure it would fit within the scope of EssentialsX.

You could consider using a more advanced sign command plugin - [ServerSigns](https://dev.bukkit.org/bukkit-plugins/serversigns/) has the ability to set global cooldowns on signs. For example, to create a sign that gives players the `boat1` kit, you can look at a sign, run `/svs add server kit boat1 <player>` (notice `<player>` is exactly what you type in, not a placeholder in this example), then `/svs setglobalcooldown 5m` to limit the sign to being used once every 5 minutes. ServerSigns has more examples of what you can do on its [wiki](https://serversigns.de/wiki/Command_Reference).
